docs/sql-ref-syntax-aux-show-create-table.md

```diff
@@ -19,4 +19,39 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+`SHOW CREATE TABLE` returns the `CREATE TABLE` statement that creates a given table. `SHOW CREATE TABLE` on a non-existent table throws Exception.
+
+### Syntax
+{% highlight sql %}
+SHOW CREATE TABLE table_name
+{% endhighlight %}
+
+### Parameters
+<dl>
+ <dt><code><em>table_name</em></code></dt>
+ <dd>The name of the table to be used for SHOW CREATE TABLE.</dd>
+</dl>
+
+### Examples
+{% highlight sql %}
+CREATE TABLE test (c INT);
```

Comment:
@huaxingao Actually i don't feel strongly about it. So you can safely ignore this comment. I was thinking if we should have this DDL with explicit, row format serde, storage properties .. so that there is less gap between input and output. What do you think ?

Reply:
I will change my example.

Reply:
I pushed the changes to remote branch. Somehow it didn't show here. :(